pandas/tests/series/methods/test_searchsorted.py

```diff
@@ -65,3 +67,10 @@ def test_searchsorted_sorter(self):
         res = ser.searchsorted([0, 3], sorter=np.argsort(ser))
         exp = np.array([0, 2], dtype=np.intp)
         tm.assert_numpy_array_equal(res, exp)
+
+    def test_searchsorted_Dataframe_fail(self):
```

Comment:
can you de-capitalize "DataFrame" here, and add a comment `# GH#49620` on the nex tline